src/praisonai/praisonai/cli/commands/realtime.py

```
@@ -27,22 +27,26 @@ def realtime_main(
+            app_dir="ui_realtime",
+            default_app_name="ui_realtime",
+            port=port,
+            host="127.0.0.1",
+            app_file=None,
+            reload=False,
+            ui_name="Realtime Voice"
+        )
+    except ImportError:
+        print('\033[91mERROR: Realtime UI is not installed.\033[0m')
+        print('Install with: pip install "praisonai[ui]"')
+        raise typer.Exit(1)
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)